types/express-serve-static-core/index.d.ts

```diff
@@ -961,7 +961,7 @@ export interface Response<
      * After middleware.init executed, Response will contain req property
      * See: express/lib/middleware/init.js
      */
-    req?: Request;
+    req: Request;
```

Comment:
this is a change for `express-serve-static-core`. did you want this included in this PR?

Reply:
Yes, otherwise the express response type incorrectly extends the request type.
I looked into the express code and it seems practically impossible to optain a response without `req` set.